cpp/include/cudf_test/type_list_utilities.hpp

```diff
@@ -42,10 +42,10 @@
  * The test `mytest` will be invoked 3 times, once for each of the types `int,
  * char, float`.
  *
- * Instead of using `::testing::Types` directly, we provide
- * `cudf::testing::Types`. This is a drop in replacement for GTest's
- * `::testing::Types`. In lieu of including `gtest/gtest.h`, include
- * `cudf_gtest.hpp` to ensure `cudf::testing::Types` is used.
+ * Instead of using \::testing::Types directly, we provide
+ * `cudf::test::Types`. This is a drop in replacement for GTest's
+ * \::testing::Types. In lieu of including `gtest/gtest.h`, include
```

Comment:
What do these backslashes `\::` do here?

Reply:
This escapes the explicit linking. doxygen takes `::` as explicit linking to Classes or functions. So, it tried to find the function/class in the documentation. `\::` is a hack to skip explicit linking.